site/docs/how-to-verify-a-release.md

```diff
@@ -0,0 +1,149 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# The Release Candidate
+
+Each Apache Iceberg release is validated by the community by holding a vote. A community release manager
+will prepare a release candidate and call a vote on the Iceberg
+[dev list](https://iceberg.apache.org/#community/#mailing-lists).
+To validate the release candidate, community members will test it out in their downstream projects and environments.
+It's recommended to report the Java, Scala, Spark, Flink and Hive versions you have tested against when you vote.
+
+In addition to testing in downstream projects, community members also check the release's signatures, checksums, and
+license documentation.
+
+## Testing the Release Candidate
```

Comment:
This should be "Testing release binaries" and should probably go after "Testing a source release candidate". The ASF technically only releases source tarballs. Binaries are all "convenience" binaries and are not quite official. We don't really need to go into detail about the distinction on this page, but the more important information is supposed to be how to verify the source tarball, then how to test with the binaries.

Reply:
This is one of those things that projects get in trouble for, so let's be sure to focus on the source based release

Reply:
Maybe it would make sense to also add a quick example of using the source release to publish a local maven artifact, and running off that.

Reply:
I agree with focusing on the source release, but I don't think we actually want people to build their own maven artifacts. That would conflict with the main artifacts that will be published.

Reply:
Updated the title to `Testing release binaries` and moved it after the `Validating a source release candidate` section. I also change `Verifying with Spark` and `Verifying with Flink` to be subsections here.